scripts/svgo.config.cjs

```diff
@@ -28,6 +30,7 @@ module.exports = {
 		},
 		{ name: 'removeTitle' },
 		{ name: 'removeComments' },
-		{ name: 'removeMetadata' }
+		{ name: 'removeMetadata' },
+		svgoViewBoxPlugin
```

Comment:
It works as expected.

![image](https://github.com/user-attachments/assets/ec74be2b-c2f4-439d-a0d7-e1bdaff8181d)

However, it turns out that the commercial repository has not executed this validator.

Fortunately, icons will land in a single repository soon so that we can deal with it.

Reply:
I'm not sure how to reproduce this issue, could you provide reproduction steps?